include/revng/ABI/Trait.h

```diff
@@ -237,9 +237,6 @@ concept TraitLike = requires(TraitType<ABI> Trait) {
   /// ARM ABI), the value of `StackAlignment` should be equal to 4.
   { Trait.StackAlignment } -> convertible_to<size_t>;
 
-  /// States the required size for a single stack argument in bytes.
-  { Trait.MinimumStackArgumentSize } -> convertible_to<size_t>;
-
```

Comment:
Can you briefly explain in the commit message why we're doing this?

Reply:
Something like this? https://github.com/revng/revng/pull/241/commits/8443c3196320618f238689ed0473481826371937